### PR TITLE
docs: auto generate pouch cli/api docs via code

### PR DESCRIPTION
**Pull request by pouchrobot**
Signed-off-by: pouchrobot <pouch-dev@alibaba-inc.com>

**1.Describe what this PR did**
This PR is automatically done by AI-based collaborating robot.
Pouchrobot will auto-generate cli/api document via https://github.com/spf13/cobra/tree/master/doc every day.
	
**2.Does this pull request fix one issue?** 
None

**3.Describe how you did it**
First, execute command "make client" to build new pouch cli;
Second, execute command "./pouch gen-doc" to generate new cli docs. 

For API part, we use a tool swagger2markup to make it.

**4.Describe how to verify it**
None

**5.Special notes for reviews**
The cli/api doc must be automatically generated.

### CODE REVIEW

File: docs/api/api.md [hunk 1]
```diff
@@ -541,7 +590,7 @@ GET "/containers/json""
 |Name|Description|Schema|
 |---|---|---|
 |**Command**  <br>*optional*||string|
-|**Created**  <br>*optional*||integer (int64)|
+|**Created**  <br>*optional*||string|
```
Comment (allencloud) [inline docs/api/api.md]: Here, we will make some change. `Create` field has a struct of string, other original type is init64.

Actually in Moby's API, consistency exists when `CreateAt` field is a type of string, but field `Create` is a type of int64. I think we should make this consistent.

In addition, from my point of view. I think int64 is better for time field in API struct. (If that, we need to update type again but worth it). If we return a string, client side cannot deal it sometimes, but if we return int64, users can deal it whatever they like.

@skoo87 WDYT?

Comment (skoo87) [reply docs/api/api.md]: I agree with your idea, string is not better than int64. But this is not compatible with moby api. Later, We may not  be able to upgrade well.

Comment (allencloud) [reply docs/api/api.md]: Yes, upgrading is a reality we must take into consideration. While at the very beginning, we also need to spend more energy in design. It is a tradeoff. But I think first thing we need to do is keep compatibility.

**Comment by allencloud** [comment]
LGTM, merging...

**Pull request merged**